fgrpc/pingsrv.go

```diff
@@ -51,9 +53,19 @@ func (s *pingSrv) Ping(c context.Context, in *PingMessage) (*PingMessage, error)
 // get a dynamic server). Pass the healthServiceName to use for the
 // grpc service name health check (or pass DefaultHealthServiceName)
 // to be marked as SERVING.
-func PingServer(port string, healthServiceName string) int {
+func PingServer(port string, secure bool, healthServiceName string) int {
 	socket, addr := fnet.Listen("grpc '"+healthServiceName+"'", port)
-	grpcServer := grpc.NewServer()
+	var opts []grpc.ServerOption
+	if secure {
+		tlsCfg, err := ftls.NewCredentials(false, ftls.DefaultServerCert, ftls.DefaultServerKey,
```

Comment:
if they are hard coded what is the point of passing constants.
we need some grpcoptions struct to control tls instead of just 1 boolean